cgo.go

```diff
@@ -0,0 +1,11 @@
+// +build !customenv
+
+package gocv
+
+/*
+#cgo !windows pkg-config: opencv
+#cgo darwin   CXXFLAGS:   --std=c++1z -stdlib=libc++
+#cgo windows  CPPFLAGS:   -IC:\opencv\build\install\include
```

Comment:
Hardcoding the paths here is a bit user-hostile IMHO and would be really tricky to debug for users.

Reply:
That is what the `env.cmd` is doing now, so we're not any worse off with this change. If you wish to build using a custom path, that would be something we can document how to do.

Reply:
I mean it's worth off in the sense that if someone upgrades their opencv version they do not have to modify the sources to be able to build. But I don't really mind personally.